source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
+      'darwin': '/png/22x22_white.png',
+      'win32': '/icon.ico'
+    }
+
+    this._mainWindow.on('show', async () => {
+      if (this._tray == null) {
+        if (process.platform === 'linux') {
+          let size = '16'
+          const sizeList = ['16', '24', '32', '48', '64', '96', '128', '256', '512']
+          const display = screen.getPrimaryDisplay()
+          if (process.env.XDG_CURRENT_DESKTOP === 'GNOME') {
+            size = display.workArea.y + ''
+          } else if (process.env.XDG_CURRENT_DESKTOP === 'KDE'){
+            size = display.size.height - display.workArea.height + ''
```

Comment:
`+ ''` is convert to string is not easily understood. It isn't used elsewhere in Zettlr code. I suggest:
```typescript
            size = (display.size.height - display.workArea.height).toString()
```

Reply:
What about the other Linux desktops? (other than Gnome or KDE). I propose the following (pseudo code). What do you think?

```
size = display.workArea.y
if (size => 8 and size <= 256)
  size is in sensible taskbar range, so use this size to determine best icon size
else
  size = display.size.height - display.workArea.height
  if (size => 8 and size <= 256)
    size is in sensible taskbar range, so use this size to determine best icon size
  else
    use fallback icon size of 32x32. A sensible fallback for most desktop setups.
```